src/stores/ChannelMapStore/ChannelMapStore.ts

```diff
@@ -64,11 +64,11 @@ export class ChannelMapStore {
         TileService.Instance.requestChannelMapTiles(tiles, this.masterFrame, midPointTileCoords, compressionQuality, {min: this.startChannel, max: this.endChannel});
     }, 100);
 
-    @action setMasterFrame(masterFrame: FrameStore) {
-        this.masterFrame = masterFrame;
+    @action setMasterFrame(masterFrame?: FrameStore) {
+        this.masterFrame = masterFrame as FrameStore;
```

Comment:
If we allow `this.masterFrame` to be undefined, please change the type of `this.masterFrame` instead:
```js
@observable masterFrame: FrameStore | undefined;

 @action setMasterFrame(masterFrame?: FrameStore) {
        this.masterFrame = masterFrame;
        ...
```